mlflow/utils/rest_utils.py

```diff
@@ -348,6 +406,12 @@ class MlflowHostCreds:
             Sets the verify param of the ``requests.request``
             function (see https://requests.readthedocs.io/en/master/api/).
             If this is set ``ignore_tls_verification`` must be false.
+        auth_by_databricks_sdk: A boolean value represent whether using Databricks SDK for
+            authentication.
```

Comment:
```suggestion
        use_databricks_sdk: A boolean value represent whether using Databricks SDK for
            authentication.
```

I'd rename this to `use_databricks_sdk`. `auth_by_databricks_sdk` sounds like we use databricks-sdk just **for authentication**.